packages/ui5-task-i18ncheck/lib/utils.js

```diff
@@ -0,0 +1,140 @@
+const xmldom = require('xmldom');
```

Comment:
would you mind porting this to ES6 please to adhere to the overall ES6 approach we're using?

Reply:
Hello Volker, do you mean the ES6 import syntax? Or the file contents e.g. use of let instead of var & fat arrow functions?

Reply:
the entire file